### PR TITLE
chore: improve logging

### PR DESCRIPTION
**Pull request by wa0x6e**
## 🧿 Current issues / What's wrong ?

Logging of what is currently happening is not standardized. We are relying on some `console.log` at some place, and almost no way to track HTTP request send to express.

The code is also using `winston` for logging, but does not seem to provide additional value.

## 💊 Fixes / Solution

- Remove `winston` package

It was added because was also used in some other repos, but does not seem to procure any values other `console.log`. Most of this PR changes come from removing this package usage.
All logging are directly collected from `yarn start` command output, and send to logtail.

- Add a more robust log system for express endpoint.

## 🚧 Changes

- Remove `winston` package, and refactor the code to use `console.log/error` instead
- Install `morgan` : morgan logs details about each incoming HTTP request, like a conventional http proxy (apache/nginx). We currently do not have anything similar, beside a `console.log('/api/xxxx')`.
- Add `serve-favicon` : middleware to serve favicon, and remove all related 404 errors from log

## 🛠️ Tests

Start an instance with `yarn dev`.

In the console output, it should log activities about the queue, as well as incoming http request (try calling any url)

### CODE REVIEW

**Comment by ChaituVR** [comment]
Agree i have confusions with winston before, pain to use it in development, maybe there is some option to print logs in dev env too?

File: src/index.ts [hunk 1]
```diff
@@ -14,6 +16,14 @@ const PORT = process.env.PORT || 3000;
 app.use(express.json({ limit: '4mb' }));
 app.use(cors({ maxAge: 86400 }));
 app.use(compression());
+app.use(
+  morgan(
+    '[http] :remote-addr - :remote-user [:date[clf]] ' +
```
Comment (ChaituVR) [inline src/index.ts:21]: We would need all these info? :)

**Review by ChaituVR — APPROVED** [review]
utAck

File: src/index.ts (hunk 1 above)
Comment (wa0x6e) [reply src/index.ts:21]: > Agree i have confusions with winston before, pain to use it in development, maybe there is some option to print logs in dev env too?

Error tracking system such as Sentry should do that. Will be on the next betting table

Comment (wa0x6e) [reply src/index.ts:21]: HTTP logs are always useful. You know who send what, when, and the response. Current logs are only printing on error, we can not track the ratio of success vs error. You can also track spammer, and weird request who overload the server

**Pull request merged**